Resources/Prototypes/DeltaV/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -256,4 +256,46 @@
     magState: mag
     steps: 5
     zeroVisible: true
-  - type: Appearance
\ No newline at end of file
+  - type: Appearance
+
+- type: entity
+  name: Decimator Laser
+  parent: BaseWeaponBattery
+  id: WeaponLaserRailgun
+  size: Huge
```

Comment:
size isn't defined in the entity section
- type: Item
  sprite: DeltaV/Objects/Weapons/Guns/Railguns/railgun_laser.rsi
  size: Huge